### PR TITLE
Resolve TODOs 04, 05, 08, 09

### PR DESCRIPTION
**Pull request by djuszczakk**

### CODE REVIEW

**Review by arkadiuszpietraszek — APPROVED** [review]
LGTM

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 1]
```diff
@@ -554,10 +554,11 @@ To read and write a value of the Bluetooth characteristic:
    var gattService = remoteDevice.getService(serviceUUIDs[0]);
    ```
 
-4. Select an interesting characteristic from the `characteristics` attribute of the `BluetoothGATTService` interface (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothGATTService), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothGATTService) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothGATTService) applications). In this example, the first characteristic is used:
+4. Select an interesting characteristic from the `characteristics` attribute of the `BluetoothGATTService` interface (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothGATTService), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothGATTService) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothGATTService) applications). The characteristic may be selected by the uuid attribute:
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md]:
```suggestion
4. Select an interesting characteristic from the `characteristics` attribute of the `BluetoothGATTService` interface (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothGATTService), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothGATTService) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothGATTService) applications). The characteristic may be selected by the `uuid` attribute:
```

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 2]
```diff
@@ -666,16 +667,16 @@ To read and write a value of the Bluetooth descriptor:
...
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md]:
```suggestion
4. Select an interesting characteristic from the `characteristics` attribute of the `BluetoothGATTService` interface (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothGATTService), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothGATTService) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothGATTService) applications). The characteristic may be selected by the `uuid` attribute:
```

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 3]
```diff
@@ -666,16 +667,16 @@ To read and write a value of the Bluetooth descriptor:
...
 
    ```
-   var characteristic = gattService.characteristics[0];
+   var characteristic = gattService.characteristics.find(characteristic => characteristic.uuid === "abcd");
    ```
 
-5. Select an interesting descriptor from the `descriptors` attribute of the `BluetoothGATTCharacteristic` interface (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothGATTCharacteristic), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothGATTCharacteristic) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothGATTCharacteristic) applications). In this example, the first descriptor is used:
+5. Select an interesting descriptor from the `descriptors` attribute of the `BluetoothGATTCharacteristic` interface (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothGATTCharacteristic), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothGATTCharacteristic) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothGATTCharacteristic) applications). The descriptor may be selected by the uuid attribute:
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md]:
```suggestion
5. Select an interesting descriptor from the `descriptors` attribute of the `BluetoothGATTCharacteristic` interface (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothGATTCharacteristic), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothGATTCharacteristic) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothGATTCharacteristic) applications). The descriptor may be selected by the `uuid` attribute:
```

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 4]
```diff
@@ -734,12 +735,57 @@ To read and write a value of the Bluetooth descriptor:
       - write about start()
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md:735]: This TODO comment can be removed.

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 5]
```diff
@@ -734,12 +735,57 @@ To read and write a value of the Bluetooth descriptor:
...
+
+To start the local GATT server:
+
+1. Retrieve `BluetoothGATTServer` object (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothGATTServer), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothGATTServer) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothGATTServer) applications) with `getGATTServer()` method of the `BluetoothManager` interface (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothManager), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothManager) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothManager) applications):
+
+```
+var server = tizen.bluetooth.getGATTServer();
+```
+
+2. Set up callbacks and start the local gatt server with `start()` method of `BluetoothGATTServer` (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothGATTServer), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothGATTServer) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothGATTServer) applications) interface:
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md]:
```suggestion
2. Set up callbacks and start the local GATT server with `start()` method of `BluetoothGATTServer` (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothGATTServer), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothGATTServer) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothGATTServer) applications) interface:
```

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 6]
```diff
@@ -734,12 +735,57 @@ To read and write a value of the Bluetooth descriptor:
...
+
+```
+var onSuccess = function() {
+   console.log("Server started successfully");
+};
+
+var onError = function(e) {
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md]:
```suggestion
var onError = function(error) {
```

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 7]
```diff
@@ -734,12 +735,57 @@ To read and write a value of the Bluetooth descriptor:
...
+   console.error("Failed to start the server, error: " + e);
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md]:
```suggestion
   console.error("Failed to start the server, error: " + error.message);
```

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 8]
```diff
@@ -734,12 +735,57 @@ To read and write a value of the Bluetooth descriptor:
...
+};
+
+server.start(onSuccess, onError);
+```
+
 <!-- END #08 -->
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md:761]: This TODO comment can be removed.

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 9]
```diff
@@ -734,12 +735,57 @@ To read and write a value of the Bluetooth descriptor:
...
 
 <!-- TODO #09
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md:763]: This TODO comment can be removed.

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 10]
```diff
@@ -734,12 +735,57 @@ To read and write a value of the Bluetooth descriptor:
...
+
+To stop the local GATT server:
+
+1. Retrieve `BluetoothGATTServer` object (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothGATTServer), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothGATTServer) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothGATTServer) applications) with `getGATTServer()` method of the `BluetoothManager` interface (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothManager), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothManager) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothManager) applications):
+
+```
+var server = tizen.bluetooth.getGATTServer();
+```
+
+2. Set up callbacks and stop the local gatt server with `stop()` method of `BluetoothGATTServer` (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothGATTServer), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothGATTServer) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothGATTServer) applications) interface:
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md]:
```suggestion
2. Set up callbacks and stop the local GATT server with `stop()` method of `BluetoothGATTServer` (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothGATTServer), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothGATTServer) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothGATTServer) applications) interface:
```

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 11]
```diff
@@ -734,12 +735,57 @@ To read and write a value of the Bluetooth descriptor:
...
+
+```
+var onSuccess = function() {
+   console.log("Server stopped successfully");
+};
+
+var onError = function(e) {
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md]:
```suggestion
var onError = function(error) {
```

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 12]
```diff
@@ -734,12 +735,57 @@ To read and write a value of the Bluetooth descriptor:
...
+   console.error("Failed to stop the server, error: " + e);
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md]:
```suggestion
   console.error("Failed to stop the server, error: " + error.message);
```

File: docs/application/web/guides/connectivity/bluetooth.md [hunk 13]
```diff
@@ -734,12 +735,57 @@ To read and write a value of the Bluetooth descriptor:
...
+};
+
+server.stop(onSuccess, onError);
+```
 <!-- END #09 -->
```
Comment (pwasowski2) [inline docs/application/web/guides/connectivity/bluetooth.md:789]: This TODO comment can be removed.

**Review by pwasowski2 — CHANGES_REQUESTED** [review]

File: docs/application/web/guides/connectivity/bluetooth.md (hunk 4 above)
Comment (djuszczakk) [reply docs/application/web/guides/connectivity/bluetooth.md:735]: All TODOs will be removed in the last commit.

File: docs/application/web/guides/connectivity/bluetooth.md (hunk 8 above)
Comment (djuszczakk) [reply docs/application/web/guides/connectivity/bluetooth.md:761]: All TODOs will be removed in the last commit.

File: docs/application/web/guides/connectivity/bluetooth.md (hunk 9 above)
Comment (djuszczakk) [reply docs/application/web/guides/connectivity/bluetooth.md:763]: All TODOs will be removed in the last commit.

File: docs/application/web/guides/connectivity/bluetooth.md (hunk 13 above)
Comment (djuszczakk) [reply docs/application/web/guides/connectivity/bluetooth.md:789]: All TODOs will be removed in the last commit.

**Review by pwasowski2 — APPROVED** [review]

**Pull request merged**